install/jakartaee/bin/sig-test-pkg-list.txt

```diff
@@ -157,7 +157,7 @@ javax.xml.bind.attachment
 javax.xml.bind.helpers
 javax.xml.bind.util
 javax.xml.registry
```

Comment:
I think that javax.xml.registry is the package for JAXR and should be removed.
Also javax.xml.registry.infomodel as well.